tests/db_functions/test_abs.py

```diff
@@ -0,0 +1,42 @@
+from decimal import Decimal
+
+from django.db.models import DecimalField
+from django.db.models.functions import Abs
+from django.test import TestCase
+
+from .models import DecimalModel, FloatModel, IntegerModel
+
+
+class AbsTests(TestCase):
+
+    def test_abs_decimal(self):
+        DecimalModel.objects.create(n1=Decimal('-0.8'), n2=Decimal('1.2'))
+        obj = DecimalModel.objects.annotate(n1_abs=Abs('n1'), n2_abs=Abs('n2')).first()
+        self.assertEqual(obj.n1_abs, -obj.n1)
+        self.assertEqual(obj.n2_abs, obj.n2)
+
+    def test_abs_float(self):
+        obj = FloatModel.objects.create(f1=-0.5, f2=12)
+        obj = FloatModel.objects.annotate(f1_abs=Abs('f1'), f2_abs=Abs('f2')).first()
+        self.assertEqual(obj.f1_abs, -obj.f1)
+        self.assertEqual(obj.f2_abs, obj.f2)
+
+    def test_abs_integer(self):
+        IntegerModel.objects.create(small=12, normal=0, big=-45)
+        obj = IntegerModel.objects.annotate(
+            small_abs=Abs('small'),
+            normal_abs=Abs('normal'),
+            big_abs=Abs('big')).first()
+        self.assertEqual(obj.small_abs, obj.small)
+        self.assertEqual(obj.normal_abs, obj.normal)
+        self.assertEqual(obj.big_abs, -obj.big)
+
+    def test_abs_transform(self):
+        try:
+            DecimalField.register_lookup(Abs, 'abs')
```

Comment:
The second argument to register_lookup() and _unregister_lookup() should be omitted as it will default to `lookup_name`.

Reply:
Thanks!